experiments/tf_trainer/tf_rnn_model.py

```diff
@@ -0,0 +1,112 @@
+"""Tensorflow Estimator implementation of RNN Model with Attention"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+
+class TFRNNModel():
+  DEFAULT_HPARAMS = tf.contrib.training.HParams(max_seq_length=300)
+
+  def __init__(self, text_feature_name, target_labels):
+    self._text_feature_name = text_feature_name
+    self._target_labels = target_labels
+
+  def estimator(self, run_config):
+    estimator = tf.estimator.Estimator(
+        model_fn=self._model_fn,
+        params=TFRNNModel.DEFAULT_HPARAMS,
+        config=run_config)
+    return estimator
+
+  def _model_fn(self, features, labels, mode, params, config):
+    inputs = features[self._text_feature_name]
+    batch_size = tf.shape(inputs)[0]
+
+    rnn_layers = [
+        tf.nn.rnn_cell.GRUCell(num_units=size, activation=tf.nn.tanh)
+        for size in [128]
```

Comment:
Let's make this a hyper-parameter the rnn_cell_sizes a hyperparameter.

Reply:
Let me defer all hyper parameter stuff. Going to go through and set up the hparams for ml engine in a later PR so will do all that then.